cli/src/main.rs

```diff
@@ -58,4 +77,17 @@ fn print_help() {
     println!("    IROHA2_CONFIG_PATH is the location of your `config.json`");
     println!("    IROHA2_GENESIS_PATH is the location of `genesis.json`");
     println!("If either of these is not provided, Iroha checks the current directory.");
+    println!(
+        "Additionally, in case of absence of both IROHA2_CONFIG_PATH and `config.json`
+in the current directory, all the variables from `config.json` should be set via the environment
+as follows:"
+    );
```

Comment:
Explain that they need to passed as a JSON string. I'd add an example of that could be run from the command line in the comments/docs.